### ISSUE TITLE
[BUG] docker run converter not working correctly on selfhosted version

### ISSUE THREAD

**Issue by devcircus** [CorentinTh/it-tools#301]
**Describe the bug**
Using the docker run command from the It-tools github homepage gives the following docker-compose file:
```yaml
version: '3.3'
services:
    run:
        ports:
            - '8080:80'
        container_name: it-tools
        image: run

```

**To Reproduce**
1. Go to the [it-tools github](https://github.com/CorentinTh/it-tools) homepage 
2. Copy the docker run command: `docker run -d -p 8080:80 --name it-tools -it corentinth/it-tools`
3. Paste in the docker run converter field
4. See the incorrect docker-compose result

**Expected behavior**
Expected to see the image name translated correctly from the docker run command

**Configuration (please complete the following information):**
 - Host Device: dell t3600
 - Host OS: Ubuntu 22.10 
 - Client Browser: Brave 1.49.120
 - Version: latest

**Comment by cgoIT** [comment]
Hi, the error comes from https://github.com/magicmark/composerize. So the issue has to be opened there. it-tools only add's the UI on top of composerize.

**Comment by devcircus** [comment]
Awesome  thanks! Feel free to close. Will follow the issue upstream.

**Comment by CorentinTh** [comment]
If I find some time, I'll maybe try to remake a lib similar to composerize, but more stable and efficient, in order to no more rely on this outdated lib

**Comment by cgoIT** [comment]
LOL. I had the same idea and just started checking some options. 😜

**Comment by CorentinTh** [comment]
Fixed by #326
